apps/vr-tests-react-components/src/stories/Skeleton.stories.tsx

```diff
@@ -47,7 +62,7 @@ storiesOf('Skeleton converged', module)
   .addStory(
     'Translucent Skeleton with rectangle',
```

Comment:
These Translucent stories don't need `includeRtl` since the color doesn't affect RTL layout.